aspnetcore/security/authorization/limitingidentitybyscheme.md

```diff
@@ -82,15 +82,27 @@ At the point of authorization, the app indicates the handler to be used. Select
 # [ASP.NET Core 2.x](#tab/aspnetcore2x)
 
 ```csharp
-[Authorize(AuthenticationSchemes = "Cookie,Bearer")]
+[Authorize(AuthenticationSchemes = AuthSchemes)]
```

Comment:
Can an attribute on a class reference a (private) constant defined in that class?

Reply:
I didn't think it was possible either, but it does compile.

Reply:
Wow. 👍